jekyll/_sass/_sidebar.scss

```diff
@@ -13,10 +13,35 @@ $sidebar-spacing-base: 5px;
   background: #F7F7F7;
   font-family: "roboto";
   height: 92vh;
-  padding: 30px;
-  overflow-y: scroll;
+  padding: 30px 24px 30px 16px;
```

Comment:
@emilmilanov In order to prevent cut off of the final item on the sidebar at certain screen sizes, we need to bump up the bottom padding here. Testing it on my local, it looked like a bottom padding of around 60px works for this purpose: `padding: 30px 24px 60px 16px`;

Reply:
Thanks Trevor. This is fixed 👍